src/services/mocks/temperatureServiceMock.js

```diff
@@ -1,4 +1,4 @@
-var DHTState = require('../dhtstate')
+var DHTState = require('../../dtos/dhtstate')
```

Comment:
We should maybe improve this, so we can reference our files relative to project root.